Package.swift

```diff
@@ -24,4 +24,3 @@ let package = Package(
     ],
     swiftLanguageVersions: [.v4, .v4_2, .v5]
 )
-let version = Version(1, 0, 1)
```

Comment:
I missed this in previous work. We don't need to set the package version in the Package.swift file – we should instead defer to git tags.